labs/lfdt/fabric-rust-integration

```diff
@@ -0,0 +1,21 @@
+---
+layout: default
+title: Hyperledger Fabric Rust Integration
+[//]: # (parent: Labs)
+---
+# Lab Name
+Hyperledger Fabric Rust Integration
+
+# Short Description
+Integration of the rust language in the hyperledger fabric ecosystem
+
+# Scope of Lab
+This lab focus on support chaincode written in rust as well as providing a library for low-level api access to hyperledger fabric peers.
```

Comment:
I understand what chaincode written in rust means.  Can you provide more details about the low-level library?

Reply:
With low level, I meant just basic functionality. No fancy stuff planned, since I want to focus on getting the basics implemented first. If the basics work, then we can talk about a more fancy integration.

Reply:
That's fine, I agree the focus should be on rust chaincode basics. I'd just ask that you clarify this sentence to make that clear and add some detail beyond one sentence. For example how do you intend to extend fabric-protos to generate a rust library that can be used by this lab?

Reply:
@denyeart I've added the plans with fabric-protos in the description. Does it sound ok or do you need more details about the implementation?

Reply:
"as well as providing a library for basic api access to hyperledger fabric peers" still is unclear, it makes it sound like you will be implementing a client SDK for peer access, which I don't think is your plan.

I'd suggest:

This lab focuses on a chaincode library written in rust so that users can write chaincode in rust. A rust binding will be added to fabric-protos to support the chaincode library.

Reply:
No, I am also planning to add api access. In my opinion, if it supports rust as chaincode, it should also support talking to the network in rust.